ui/src/components/ServiceInstancesView.tsx

```diff
@@ -45,40 +43,25 @@ function ServiceInstancesView(props: any) {
       setSecret(secret);
       if (Ok(secret)) {
         axios
-          .get<ServiceOfferings>(
-            api(`service-offerings/${secret.namespace}/${secret.name}`), {
-              params: {
-                sm_secret_name: secret.name,
-                sm_secret_namespace: secret.namespace
-              }
+          .get<ServiceInstances>(api("service-instances"), {
+            params: {
+              sm_secret_name: secret.name,
+              sm_secret_namespace: secret.namespace
             }
-          )
+          })
           .then((response) => {
-            setOfferings(response.data);
-            axios
-              .get<ServiceInstances>(api("service-instances"), {
-                params: {
-                  sm_secret_name: secret.name,
-                  sm_secret_namespace: secret.namespace
-                }
-              })
-              .then((response) => {
-                setServiceInstances(response.data);
-                setError(null);
-                if (id) {
-                  const instance = response.data.items.find((instance) => instance.id === id);
-                  if (instance) {
-                    openPortal(instance);
-                  }
-                }
-                setLoading(false);
-              })
-              .catch((error) => {
-                setLoading(false);
-                setError(error);
-              });
+            setServiceInstances(response.data);
+            setError(null);
+            if (id) {
+              const instance = response.data.items.find((instance) => instance.id === id);
+              if (instance) {
+                openPortal(instance);
+              }
+            }
+            setLoading(false);
           })
           .catch((error) => {
+            setServiceInstances(error);
```

Comment:
Setting error instances is not correct. Error is not a list of service instances.

Reply:
Changed to undefined.